### PR TITLE
Update the content of Package.md and README.md file and Fix the icon res issue

### PR DESCRIPTION
**Pull request by ayeshLK**
Related to [#2657](https://github.com/ballerina-platform/ballerina-standard-library/issues/2657)

## Examples
N/A

## Checklist
- [X] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: ballerina/Module.md [hunk 1]
```diff
@@ -67,19 +67,16 @@ WebSub specification extensively discusses the relationship between the subscrib
...
+websubhub:TopicDeregistrationSuccess topicRegResponse = check publisherClient->registerTopic("http://websubpubtopic.com");
+json payload = {
+    "action": "publish",
+    "mode": "remote-hub"
+};
+websubhub:Acknowledgement publishResponse = check publisherClient->publishUpdate("http://websubpubtopic.com", payload);
 ```
 
-# Returning Errors from Remote Methods
+### Returning Errors from Remote Methods
```
Comment (ayeshLK) [inline ballerina/Module.md:79]:
```suggestion
#### Returning Errors from Remote Methods
```

File: ballerina/Package.md [hunk 2]
```diff
@@ -1,8 +1,12 @@
...
 
-The `websubhub` package is one of the standard library packages of the <a target="_blank" href="https://ballerina.io/">Ballerina</a> language.
+This package provides APIs for a WebSub Hub service and WebSub Publisher client.
 
-This package provides the capability to easily implement a WebSub compliant **Hub Service** and **Publisher Client**.
+[**WebSub**](https://www.w3.org/TR/websub/) is a common mechanism for communication between publishers of any kind of web content and their subscribers based on HTTP webhooks. Subscription requests are relayed through hubs, which validate and verify the requests. Hubs then distribute new and updated content to subscribers when it becomes available. WebSub was previously known as PubSubHubbub.
+
+[**WebSub Hub**](https://www.w3.org/TR/websub/#hub) is an implementation that handles subscription requests and distributes the content to subscribers when the corresponding topic URL has been updated.
+
+[**WebSub Publisher**](https://www.w3.org/TR/websub/#publisher) is an implementation that advertises a topic and hub URL on one or more resource URLs.
```
Comment (daneshk) [inline ballerina/Package.md:9]: Shall we add sub-sections as well with code snippets?

Comment (ayeshLK) [reply ballerina/Package.md:9]: Fixed with [e9db73c302362b33e70a1a7706ad50620f8cc12a](https://github.com/ballerina-platform/module-ballerina-websubhub/pull/434/commits/e9db73c302362b33e70a1a7706ad50620f8cc12a)

File: README.md [hunk 3]
```diff
@@ -7,11 +7,121 @@ Ballerina WebSubHub Library
...
+This module provides APIs for a WebSub Hub service and WebSub Publisher client.
 
-This package provides the capability to easily implement a WebSub compliant **Hub Service**.
+[**WebSub**](https://www.w3.org/TR/websub/) is a common mechanism for communication between publishers of any kind of web content and their subscribers based on HTTP webhooks. Subscription requests are relayed through hubs, which validate and verify the requests. Hubs then distribute new and updated content to subscribers when it becomes available. WebSub was previously known as PubSubHubbub.
+
+[**WebSub Hub**](https://www.w3.org/TR/websub/#hub) is an implementation that handles subscription requests and distributes the content to subscribers when the corresponding topic URL has been updated.
+
+[**WebSub Publisher**](https://www.w3.org/TR/websub/#publisher) is an implementation that advertises a topic and hub URL on one or more resource URLs.
 
  For more information, go to [The WebSub Hub Module](https://ballerina.io/learn/api-docs/ballerina/websubhub/index.html).
```
Comment (daneshk) [inline README.md:18]:
```suggestion
```

Comment (daneshk) [reply README.md:18]: Let's remove this section

File: README.md [hunk 4]
```diff
@@ -7,11 +7,120 @@ Ballerina WebSubHub Library
   [![Github issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-standard-library/module/websubhub.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Fwebsubhub)
   [![codecov](https://codecov.io/gh/ballerina-platform/module-ballerina-websubhub/branch/main/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerina-websubhub) 
 
-The WebSubHub package is one of the standard library packages of the <a target="_blank" href="https://ballerina.io/">Ballerina</a> language.
+This module provides APIs for a WebSub Hub service and WebSub Publisher client.
```
Comment (ayeshLK) [inline README.md:10]:
```suggestion
This library provides APIs for a WebSub Hub service and WebSub Publisher client.
```

Comment (daneshk) [inline README.md:10]:
```suggestion
This library provides APIs for a WebSub Hub service and WebSub Publisher client.
```

**Review by daneshk — APPROVED** [review]
LGTM

File: README.md [hunk 5]
```diff
@@ -15,7 +15,10 @@ This library provides APIs for a WebSub Hub service and WebSub Publisher client.
 
 [**WebSub Publisher**](https://www.w3.org/TR/websub/#publisher) is an implementation that advertises a topic and hub URL on one or more resource URLs.
 
+<<<<<<< Updated upstream
 
+=======
+>>>>>>> Stashed changes
```
[the remark this answers is not in the document]
Comment (ayeshLK) [reply README.md]: Fixed with [300f121560cf2eed7c855b2153dfa20dd3015f9d](https://github.com/ballerina-platform/module-ballerina-websubhub/pull/434/commits/300f121560cf2eed7c855b2153dfa20dd3015f9d)

**Review by shafreenAnfar — APPROVED** [review]

**Pull request merged**